### PR TITLE
Updating to build '20220222.7' of 'https://github.com/dotnet/arcade'

### PR DESCRIPTION
**Pull request by ericstj**
Darc update of ".NET 6 Eng" channel

### CODE REVIEW

File: eng/common/dotnet-install.sh [hunk 1]
```diff
@@ -70,7 +70,7 @@ case $cpuname in
     ;;
 esac
 
-dotnetRoot="$repo_root/.dotnet"
+dotnetRoot="${repo_root}.dotnet"
```
Comment (tarekgh) [inline eng/common/dotnet-install.sh:73]: >repo_root [](http://example.com/codeflow?start=14&length=9)

does the repo_root always appended with the path separator?

File: eng/common/generate-sbom-prep.ps1 [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+Write-Host "Creating dir $ManifestDirPath"
+# create directory for sbom manifest to be placed
+if (!(Test-Path -path $ManifestDirPath))
+{
+  New-Item -ItemType Directory -path $ManifestDirPath
+  Write-Host "Successfully created directory $ManifestDirPath"
+}
+else{
+  Write-PipelineTelemetryError -category 'Build'  "Unable to create sbom folder."
```
Comment (tarekgh) [inline eng/common/generate-sbom-prep.ps1:13]: >Write-PipelineTelemetryError -category 'Build'  "Unable to create sbom folder." [](http://example.com/codeflow?start=2&length=79)

Is it possible the folder already created?

File: eng/common/dotnet-install.sh [hunk 3]
```diff
@@ -70,7 +70,7 @@ case $cpuname in
...
 if [[ $architecture != "" ]] && [[ $architecture != $buildarch ]]; then
   dotnetRoot="$dotnetRoot/$architecture"
```
Comment (tarekgh) [inline eng/common/dotnet-install.sh:75]: >"$dotnetRoot/$architecture" [](http://example.com/codeflow?start=13&length=27)

worth it to use join-path?

Comment (ericstj) [reply eng/common/dotnet-install.sh:75]: Everything in eng/common is just copied from arcade by darc.  We don't need to review it -- or reviews should be done in dotnet/arcade before it flows to consumers.

File: eng/common/generate-sbom-prep.ps1 (hunk 2 above)
Comment (ericstj) [reply eng/common/generate-sbom-prep.ps1:13]: You can share this feedback here: https://github.com/dotnet/arcade/pull/8479

File: eng/common/dotnet-install.sh (hunk 1 above)
Comment (ericstj) [reply eng/common/dotnet-install.sh:73]: Feedback for https://github.com/dotnet/arcade/pull/7498

**Review by michaelgsharp — APPROVED** [review]

**Comment by ericstj** [comment]
Thanks for merging this, @michaelgsharp

**Pull request merged**